protocol/daemons/liquidation/client/client.go

```diff
@@ -75,60 +119,105 @@ func Start(
 		); err != nil {
 			// TODO(DEC-947): Move daemon shutdown to application.
 			logger.Error("Liquidations daemon returned error", "error", err)
+			client.RecordUpdateFailure(timeProvider, err)
+		} else {
+			client.RecordUpdateSuccess(timeProvider)
 		}
 	}
+}
 
-	return nil
+// CheckCollateralizationForSubaccounts queries a gRPC server using `AreSubaccountsLiquidatable`
```

Comment:
There are no actual changes to the following methods. I just removed `RunLiquidationDaemonTaskLoop` from this file into `sub_task_runner.go` as a method on the `SubTaskRunner` interface. This is the same design as the pricefeed daemon and allows me to to mock the `RunLiquidationDaemonTaskLoop` method in testing.